packages/expo-updates/ios/EXUpdates.podspec

```diff
@@ -30,17 +30,20 @@ Pod::Spec.new do |s|
     'SWIFT_COMPILATION_MODE' => 'wholemodule'
   }
 
-  if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("#{s.name}.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
+  ex_updates_native_debug = ENV['EX_UPDATES_NATIVE_DEBUG'] == '1'
```

Comment:
i have some questions for the implementations. if this is only for expo-updates, why should we change expo, expo-modules-core, and autolinking?

- podspec supports [`compiler_flags`](https://guides.cocoapods.org/syntax/podspec.html#compiler_flags) to add defines, so we don't have to do it through autolinking.
- we could also add `pod_target_xcconfig.OTHER_SWIFT_FLAGS` if the definitions should be in swift code.

i hope we could scope the changes inside expo-updates and keeps it independent from other modules.

Reply:
I will attempt to scope the changes to expo-updates only, and see if I can make that work.